google/cloud/monitoring_v3/gapic/transports/service_monitoring_service_grpc_transport.py

```diff
@@ -183,7 +226,37 @@ def delete_service(self):
     def create_service_level_objective(self):
         """Return the gRPC stub for :meth:`ServiceMonitoringServiceClient.create_service_level_objective`.
 
-        Create a ``ServiceLevelObjective`` for the given ``Service``.
+        Protocol Buffers - Google's data interchange format Copyright 2008
```

Comment:
Something strange is happening. This is not coming from the protos. 

https://github.com/googleapis/googleapis/blob/b21bc9b46e3263cd4ebe50f60c61589e3fe797f2/google/monitoring/v3/service_service.proto#L88

Reply:
This is super weird. I can't reproduce it. I've just regenerated monitoring client and it looks ok. There are some minor formatting differences in the documentation, but nothing even close to the total mess happening in this PR.

This looks like the whole mapping between documentation and places where it should be inserted was competely messed up. 

@busunkim96 Can you reproduce this on your local machine?

Reply:
It goes away when I run it locally. 🤔 I'm going to hope this was a fluke and see what happens tomorrow morning.